lisa/features/disks.py

```diff
@@ -64,6 +64,17 @@ def _initialize(self, *args: Any, **kwargs: Any) -> None:
     def get_resource_disk_mount_point(self) -> str:
         raise NotImplementedError
 
+    # Check if OS disk is of NVMe type to confirm ASAP enablement.
+    # Incase of ASAP VMs, disk controller is NVMe.
+    # Even non NVMe disks are treated as NVMe.
+    @property
+    def is_asap_enabled(self) -> bool:
```

Comment:
Is the "asap" a azure specified term? If so, it should be changed. And the logic is not used widely, it can be written in test case level.

Reply:
ASAP is azure specific name. Changed the name to more generic name 'disk_controller_type',